vignettes/quickstart.Rmd

```diff
@@ -176,22 +161,19 @@ models and the references to use when imputing the missing values. It's usage is
 ```{r}
 imputeObj <- impute(
     drawObj,
-    c("A" = "A", "B" = "A")
+    c("DRUG" = "PLACEBO", "PLACEBO" = "PLACEBO")
 )
 imputeObj
 ```
 
-In this instance we are specifying that group `A` should use itself as its reference group
-(this is commonly done in the case of a placebo or control arm) and that group `B` should
-use group `A` as its reference group. 
+In this instance we are specifying that group `PLACEBO` should use itself as its reference group and that group `DRUG` should
+use group `PLACEBO` as its reference group (as standard for imputation using reference-based methods). 
```

Comment:
group `DRUG` should use group `PLACEBO` -> group `DRUG` should use **the** group `PLACEBO`

Reply:
done